web-local/src/lib/components/workspace/explore/time-controls/TimeControls.svelte

```diff
@@ -148,15 +161,25 @@ Constructs a TimeRange object – to be used as the filter in MetricsExplorer 
 </script>
 
 <div class="flex flex-row">
-  <TimeRangeNameSelector
-    {allTimeRange}
-    {metricViewName}
-    on:select-time-range-name={setSelectedTimeRangeName}
-    {selectedTimeRangeName}
-  />
-  <TimeGrainSelector
-    on:select-time-grain={setSelectedTimeGrain}
-    {selectableTimeGrains}
-    {selectedTimeGrain}
-  />
+  {#if !hasTimeSeries}
+    <div
+      on:click={() => goto(`/dashboard/${metricViewName}/edit`)}
+      class="px-3 py-2 flex flex-row items-center gap-x-3 cursor-pointer"
+    >
+      <span class="ui-copy-icon"><Calendar size="16px" /></span>
+      <span class="italic ui-copy-disabled">No time dimension specified</span>
```

Comment:
```suggestion
      <span class="ui-copy-disabled">No time dimension specified</span>
```

The italic looks out of place here, especially when juxtaposed with the "no filters selected" below it. Should suffice to just remove that